config/config.go

```diff
@@ -0,0 +1,150 @@
+// Copyright © 2018 Developer developer@vamp.io
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"github.com/magneticio/vampkubistcli/logging"
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+	yaml "gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// ClientConfig respresents serializable client configuration
+type ClientConfig struct {
+	Url            string `yaml:"url,omitempty" json:"url,omitempty"`
+	Cert           string `yaml:"cert,omitempty" json:"cert,omitempty"`
+	Username       string `yaml:"username,omitempty" json:"username,omitempty"`
+	RefreshToken   string `yaml:"refresh_token,omitempty" json:"refresh_token,omitempty"`
+	AccessToken    string `yaml:"access_token,omitempty" json:"access_token,omitempty"`
+	ExpirationTime int64  `yaml:"expiration_time,omitempty" json:"expiration_time,omitempty"`
+	Project        string `yaml:"project,omitempty" json:"project,omitempty"`
+	Cluster        string `yaml:"cluster,omitempty" json:"cluster,omitempty"`
+	VirtualCluster string `yaml:"virtualcluster,omitempty" json:"virtualcluster,omitempty"`
+	APIVersion     string `yaml:"apiversion,omitempty" json:"apiversion,omitempty"`
+}
+
+//var Config ClientConfig
+
+// RestClientConfig provides fields and methods for client configuration manipulation
+type RestClientConfig struct {
```

Comment:
This should not be here. Please split RestClient config and command line client Config.
We are keeping RestClient Config separately because it will be used in other projects.
That is why we have two configurations with similar structures we can not merge them.